helm_deploy/values-dev.yaml

```diff
@@ -5,7 +5,8 @@ generic-service:
   replicaCount: 2
 
   ingress:
-    host: welcome-dev.prison.service.justice.gov.uk
+    hosts:
+      - welcome-dev.prison.service.justice.gov.uk
```

Comment:
Not required, but recommended for v2 of generic service https://github.com/ministryofjustice/hmpps-helm-charts/releases/tag/generic-service-2.0.0